rubi/rubi/client.py

```diff
@@ -78,6 +80,7 @@ def __init__(
             network=self.network, wallet=self.wallet, key=self.key
         )
 
+        # TODO a Dict of { symbol: ERC20 }, investigate
```

Comment:
this makes a lot of sense in the client, semantically we should probably keep addresses abstracted away